unpackaged/config/rd2_elevate_post_config/package.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
+        <members>NPSP_Recurring_Donation</members>
+        <name>FlexiPage</name>
+    </types>
+    <version>48.0</version>
```

Comment:
I recommend using v51

Reply:
@force2b I agree about considering wider application for this in the future.  It would be nice, for example, to take the config information we generate from the transport org, save it in .json format or some such, and use it in a flow to fully configured the connect Elevate org with remote site settings, etc.